spec/models/tenant_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Tenant, type: :model do
+
+  it 'is valid with name, phone, website and logo' do
+     tenant = Tenant.new(
+       name:    'Heroes co.',
+       phone:   '1-800-291-9249',
+       website: 'heroes.com',
+       logo:    ''
+    )
+    expect(tenant).to be_valid
+  end
+
+  # it { should have_many(:users).dependent(:destroy) }
+  it { should validate_presence_of(:name) }
```

Comment:
do not test validations, believe me, active record already tested them.
instead of that please test that you have valid factory. That one is much more useful.